onnxruntime/core/session/inference_session.cc

```diff
@@ -2226,6 +2229,14 @@ common::Status InferenceSession::Initialize() {
       }
 #endif
 
+#ifdef USE_QNN
+      // Register QNN EP specific GraphTransformers
+      auto transformers = GraphTransformerHelpers::GetGraphTransformers();
```

Comment:
>auto transformers = GraphTransformerHelpers::GetGraphTransformers(); [](http://example.com/codeflow?start=6&length=68)

It would be much better if this code had some naming with QNN in it or be a separate namespace.